cmd/test-utils_test.go

```diff
@@ -90,15 +89,15 @@ func init() {
 	globalIsDistErasure = false
 
 	// Disable printing console messages during tests.
-	color.Output = ioutil.Discard
+	// color.Output = ioutil.Discard
```

Comment:
Is this due to WIP or do you plan to check this in?

Reply:
TBH I find it extremely frustrating that debug logging is killed. You know what let's tie it up to `testing.Verbose`.